packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -88,7 +88,6 @@ def test_dcos_diagnostics_health(dcos_api_session):
 
 
 @retrying.retry(wait_fixed=2000, stop_max_delay=LATENCY * 1000)
-@pytest.mark.xfail("config.getoption('--windows-only')", strict=True, reason="D2IQ-64753")
 def test_dcos_diagnostics_nodes(dcos_api_session):
```

Comment:
This one fails because the IPs are not set. It relates to @janisz findings.

Reply:
Can you elaborate?

Reply:
Shouldn't we add supportedwindows anotation on all tests in this file?